isthmus/src/test/java/io/substrait/isthmus/CalciteLiteralTest.java

```diff
@@ -194,6 +194,29 @@ void tIntervalYearMonthWithPrecision() {
         convertedRex.getValueAs(BigDecimal.class).longValue());
   }
 
+  @Test
+  void tIntervalDaySecond() {
+    // Calcite stores milliseconds since Epoch, so test only millisecond precision
+    BigDecimal bd =
```

Comment:
minor: could you name this something like `milliseconds`, mostly to make the unit involved crystal-clear (it took me a bit longer that I would like to admit to figure that out from the code).